Woof/WoofTests/Model/Preferences/RoleTests.swift

```diff
@@ -0,0 +1,9 @@
+import XCTest
+
+final class RoleTests: XCTestCase {
```

Comment:
Does it make sense to remake the test in such a way as to check cases or, in principle, does the original version check for the presence of cases?

```suggestion
 final class RoleTests: XCTestCase {
    func testAllRolesExistInAPI() {
        Role.allCases.forEach { role in
            XCTAssertTrue(checkRoleExistsInAPI(role))
        }
    }

    func checkRoleExistsInAPI(_ role: Role) -> Bool {
        switch role {
        case .owner,
             .sitter,
             .none:
            return true
        }
    }
 }
```

Reply:
I agree, that my code does nothing 🤦‍♀️ 
Fixed with https://github.com/ios-course/ironfoudation-team-project/pull/111/commits/0970ea72227e9b020965214b63f454899ec4556c